app/guid-file/template.hbs

```diff
@@ -196,32 +196,24 @@
                                     </div>
                                 {{/unless}}
                                 <dl>
-                                    <dt>{{t 'file_detail.title'}}</dt>
+                                    <dt data-test-target-title-label>{{t 'file_detail.title'}}</dt>
                                     <dd data-test-target-title>{{manager.target.title}}</dd>
-                                    <dt>{{t 'file_detail.description'}}</dt>
+                                    <dt data-test-target-description-label>{{t 'file_detail.description'}}</dt>
                                     <dd data-test-target-description>{{manager.target.description}}</dd>
-                                    {{#unless manager.isAnonymous}}
-                                        <dt>{{t 'file_detail.contributors'}}</dt>
-                                        <ContributorList
-                                            data-test-target-contributors
-                                            local-class='contributor-list'
-                                            @model={{manager.target}}
-                                            @shouldLinkUsers={{true}}
-                                            @shouldTruncate={{false}}
-                                        />
-                                        <dt local-class='affiliated-institutions'>{{t 'file_detail.institutions'}}</dt>
-                                        <div data-test-target-institutions>
-                                            {{#each this.targetInstitutions as |institution|}}
-                                                <dd>{{institution.name}}</dd>
-                                            {{/each}}
-                                        </div>
-                                    {{/unless}}
-                                    <dt>{{t 'file_detail.license'}}</dt>
-                                    <dd data-test-target-license-name>{{manager.targetLicense.name}}</dd>
-                                    <dt>{{t 'file_detail.resource_type_general'}}</dt>
-                                    <dd data-test-target-resource-type>{{manager.targetMetadata.resourceTypeGeneral}}</dd>
-                                    <dt>{{t 'file_detail.language'}}</dt>
-                                    <dd data-test-target-language>{{manager.targetLanguageFromCode}}</dd>
```

Comment:
These fields need to be added bck in too

Reply:
Got it, thank you.